### PR TITLE
Move loading of credentials into a function

### PR DESCRIPTION
**Pull request by phosei**
This allows call sites to do something like this:

    bwtv.load_site_credentials(site)
    self.username = bwtv.cached_credentials.get(site)[0]
    self.password = bwtv.cached_credentials.get(site)[1]

### CODE REVIEW

**Comment by fkusei** [comment]
Lines 63/66 reference secret_id, which is not present in `load_site_credentials()`

File: bwtv.py [hunk 1]
```diff
@@ -30,20 +30,8 @@
...
-    session = sessions.setdefault(getpid(), Session())
-
-    try:
-        full_url = "{}/api/secrets/{}/".format(
-            config.get(site, "url"),
-            secret_id,
-        )
-
         if site not in cached_credentials:
             if (
```
Comment (phosei) [inline bwtv.py:40]: This `if` is strange, btw. `and` has higher priority than `or`, doesn’t it? I think the first line was meant to be put in parens?

**Comment by phosei** [comment]
> Lines 63/66 reference secret_id, which is not present in `load_site_credentials()`

Thanks, will fix tomorrow.

File: bwtv.py (hunk 1 above)
Comment (fkusei) [reply bwtv.py:40]: yes

**Pull request merged**